supplementary_style_guide/style_guidelines/specific-doc-types.adoc

```diff
@@ -5,5 +5,115 @@
+----
+
+[discrete]
+==== Known issue
+
+Use present tense for the issue and the imperative form for the workaround in the following format:
+----
+There is currently a known issue <present tense explanation> under <X conditions>. 
+
+Workaround: <workaround>.
+----
+.Example doc text
+----
+Currently, you cannot use Orchestration (heat) templates with the director to deploy an overcloud that requires NFS as an Image service (glance) back end. There is currently no workaround for this issue.
+----
+----
+The Compute services (nova) might fail to deploy because the `nova_wait_for_compute_service` script is unable to query the Nova API. If a remote container image registry is used outside of the undercloud, the Nova API service might not finish deploying in time. 
+Workaround: Rerun the deployment command, or use a local container image registry on the undercloud.
+----
+
+[discrete]
+==== Technology Preview
+
+Use present tense in the following format:
+----
+In <release>, a Technology Preview is available for <feature explanation>.
+----
+.Example doc text 
+----
+In Red Hat OpenStack Platform 16.0, a Technology Preview is available for the Bare Metal Provisioning service (ironic) deployed on an IPv6 provisioning network for BMaaS (Bare Metal as-a-Service) tenants.
+----
+
+
+[discrete]
+==== Deprecated functionality
+Warn users about the following deprecation stages:
+
+* Plan to deprecate
+* Deprecate
+* Plan to remove
+* Remove 
+
+If available, inform users of alternative capabilities and workarounds.
+
+[discrete]
+===== Deprecation notice
+----
+In <product> <release>, <name of capability> is deprecated. Bug fixes and support are provided through the end of the <releasename or, if unknown, “a future”> life cycle. After which, no new feature enhancements are made. (Optional: You can use <alternative capability> instead.) 
+----
+.Example deprecation notice doc text
+----
+In Red Hat Openstack Platform (RHOSP) 14, the director graphical user interface is deprecated. Bug fixes and support is provided through the end of a future life cycle but Red Hat do not implement new feature enhancements.
+----
+
+[discrete]
+===== Removal notice
+----
+In <product> <release>, <name of capability> is removed. Bug fixes and support is provided through the end of the <releasename or, if unknown, “a future”> life cycle. After which, no new feature enhancements are made. (Optional: You can use <alternative capability> instead.) 
+----
+.Example removal notice doc text
+----
+In Red Hat OpenStack Platform (RHOSP) 16, the Data Processing service (sahara) is removed. Bug fixes and support are provided through the end of the RHOSP 15 life cycle but Red Hat do not implement new feature enhancements.
+----
+
+
+
+
+
 [[tech-preview-features]]
```

Comment:
Hi Naomi (@ndeevy),
It looks like "Technology Preview features" is an orphaned topic and either needs to be developed or removed.
Thanks,
--Greg

Reply:
Thanks Greg, removed!